src/main/java/org/reaktivity/nukleus/http/internal/stream/ClientConnectReplyStream.java

```diff
@@ -945,11 +936,9 @@ private void handleThrottleAfterBegin(
     private void handleBoundedWindow(
         WindowFW window)
     {
-        final int targetWindowBytesDelta = window.update();
-        final int targetWindowFramesDelta = window.frames();
+        final int targetWindowBytesDelta = window.credit();
```

Comment:
Let's flush through the renaming more fully.
eg. `xxxWindowBytesDelta` => `xxx[Window]Credit`
eg. `xxxWindowBytes` => `xxx[Window]Budget`

All references to `windowBytes` should be gone.